db/seeds.rb

```diff
@@ -122,7 +122,7 @@ def create_user_contribution(user:, repository:, date:)
       contrib.company_id = user.company.id
       contrib.repository = repository
       contrib.created_at = date
-      contrib.approve
+      contrib.approve(user.id)
```

Comment:
Another option would be leaving seeded contributions at their initial state, but I was concerned about breaking tests.

Reply:
the seeds file is not used in the tests

Reply:
Oh, ok. I'll remove this to a fix PR

Reply:
there's no need, is just to let you know that the seed are not loaded in the tests